### PR TITLE
Added section on morning announcements

### PR DESCRIPTION
**Pull request by selynna**
A general description for #363

### CODE REVIEW

**Comment by selynna** [comment]
@zachlatta Noticed that we weren't supposed to link things up in the very beginning, is it okay if I added my section there?

File: cohort_4/playbook/ACTIVITIES.md [hunk 1]
```diff
@@ -549,6 +555,57 @@ elevators with large groups can get pretty chaotic, especially when going to the
...
+access for each person in your leader group.  The ability to share and 
+collaborate with your team members is ultimately beneficial as it should reduce
+the number of mistakes and incompatible pairs.
+
+## Compatibility and Uniqueness
+Pairings will not work out if the partners are not compatible.  Skill is 
+something to keep in mind, of course, but it is not necessarily the one thing
+that pairings depend on.  If a person was sociable and communicated well, but 
+did not have much experience coding, a pairing with a more advanced person 
+would still be possible because the more advanced person gets teaching experience,
```
Comment (maxwofford) [inline cohort_4/playbook/ACTIVITIES.md:599]: Would you please wrap this to 80 chars?

File: cohort_4/playbook/ACTIVITIES.md [hunk 2]
```diff
@@ -549,6 +555,57 @@ elevators with large groups can get pretty chaotic, especially when going to the
...
+
+# Pairing Guidelines
+
+Since there are many ways you can pair people in your club/camp, there are 
+several guidelines that should be followed.  Organization of this should 
+be maintained in a spreadsheet, and partners should be purposely paired
+for maximum efficiency and ideal workability.
+
+## Spreadsheet Creation
+You should create a Google Spreadsheet to maintain organization for pairings.
```
Comment (maxwofford) [inline cohort_4/playbook/ACTIVITIES.md:588]: Make sure to include a newline after every header.

File: cohort_4/playbook/ACTIVITIES.md [hunk 3]
```diff
@@ -549,6 +555,57 @@ elevators with large groups can get pretty chaotic, especially when going to the
...
+for maximum efficiency and ideal workability.
+
+## Spreadsheet Creation
+You should create a Google Spreadsheet to maintain organization for pairings.  
+Spreadsheets allow one to cleanly see each pair, and provide a means of 
+access for each person in your leader group.  The ability to share and 
+collaborate with your team members is ultimately beneficial as it should reduce
+the number of mistakes and incompatible pairs.
+
+## Compatibility and Uniqueness
```
Comment (maxwofford) [inline cohort_4/playbook/ACTIVITIES.md:594]: Newline here too

**Comment by maxwofford** [comment]
@ssun098 Thanks for this PR. Idk about the original thing about linking stuff, but this is a great addition to the playbook. Respond to comments and I'll be happy to merge!

**Comment by zachlatta** [comment]
Agreed with @MaxWofford's comments. Make sure to follow our markdown style guide (linked in https://github.com/hackedu/hackedu/blob/master/CONTRIBUTING.md).

@ssun098 it's totally cool to add a link to this to the table of contents. There's a program that we use to generate the table of contents, but you can also manually add it :-).

**Comment by zachlatta** [comment]
I'll go ahead and cherry pick into master and make the modifications @MaxWofford is talking about.

**Comment by zachlatta** [comment]
Cherry picked onto master!

**Comment by selynna** [comment]
@zachlatta thanks!

**Pull request closed without merging**